cmd/git/main.go

```diff
@@ -16,6 +16,7 @@ import (
 	"regexp"
 	"strings"
 
+	git2 "github.com/shipwright-io/build/pkg/git"
```

Comment:
This could probably have a better alias that relates to where it is coming from, like `shpgit`